mix.exs

```diff
@@ -36,10 +36,29 @@ defmodule Membrane.Template.Mixfile do
 
   defp deps do
     [
-      {:membrane_core, "~> 0.7.0"},
-      {:ex_doc, "~> 0.24", only: :dev, runtime: false},
+      {:membrane_core, "~> 0.10.0"},
+      {:bimap, "~> 1.2"},
+      {:membrane_h264_format, "~> 0.3"},
+      {:membrane_vp8_format, "~> 0.4.0"},
+      {:membrane_vp9_format, "~> 0.4.0"},
+      {:membrane_opus_format, "~> 0.3.0"},
+      {:membrane_mp4_format, "~> 0.7.0"},
+      {:membrane_common_c, "~> 0.13.0"},
+      {:membrane_file_plugin, "~> 0.12.0", runtime: false},
+      # Test dependencies
+      {:membrane_opus_plugin, "~> 0.15.0", only: :test, runtime: false},
+      {:membrane_flv_plugin, "~> 0.2.0", only: :test, runtime: false},
+      {:membrane_mp4_plugin, "~> 0.15.0", only: :test, runtime: false},
+      {:membrane_ivf_plugin, "~> 0.4.1", only: :test, runtime: false},
+      {:membrane_ogg_plugin,
+       github: "membraneframework/membrane_ogg_plugin", only: :test, runtime: false},
+      {:membrane_h264_ffmpeg_plugin, "~> 0.21.0", only: :test, runtime: false},
+      {:membrane_ffmpeg_swresample_plugin, "~> 0.15.0", only: :test, runtime: false},
+      # Credo
+      {:ex_doc, "~> 0.26", only: :dev, runtime: false},
       {:dialyxir, "~> 1.1", only: :dev, runtime: false},
-      {:credo, "~> 1.5", only: :dev, runtime: false}
+      {:credo, "~> 1.5", only: :dev, runtime: false, override: true},
```

Comment:
Override? Is it really necessary? Also, credo 1.6 for compatibility with elixir 1.13. It should be pulled automatically but better safe then sorry
```suggestion
      {:credo, "~> 1.6", only: :dev, runtime: false, override: true},
```